src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -31,12 +31,18 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
     ];
 
     this.modal.style.cssText = modalStyles.join(';');
-    this.modal.innerHTML = '<h5 style="margin-top: 20px"></h5><button style="margin:5px auto 5px">Retry?</button><p>Alternatively, <a href>reload</a></p>';
+    this.modal.innerHTML = '<h5 style="margin-top: 20px"></h5><button style="margin:5px auto 5px">Retry</button><p>Alternatively, <a href>reload</a></p>';
     this.message = this.modal.querySelector('h5')!;
     this.button = this.modal.querySelector('button')!;
     this.reloadParagraph = this.modal.querySelector('p')!;
 
-    this.button.addEventListener('click', () => window['Blazor'].reconnect());
+    this.button.addEventListener('click', async () => {
+      this.show();
+      const successful = await window['Blazor'].reconnect();
```

Comment:
Should we wrap this in a `try-catch`?

Reply:
We could but I feel like that'd hide other errors. It technically shouldn't throw.